modules/installation-mirror-repository.adoc

```diff
@@ -48,13 +87,51 @@ the pull secret for your mirror registry that you created.
 <6> The release mirror. For a production release, you must specify
 `ocp-release`.
 
+. Build the signature for the content that you plan to mirror and verify that it
+is signed by an official Red Hat key:
+.. Set the digest for the version to install to:
++
+----
+$ DIGEST="$(oc adm release info quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE} | sed -n 's/Pull From: //p')" <1>
+----
+.. Build a signature URI:
++
+----
+$ URI="https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/${DIGEST/:/=}/<signature_name>" <1>
+----
+<1> Specify the name of the signature to store.
+
+.. Download the signature:
++
+----
+$ wget "${URI}"
+----
+
+.. Verify the signature:
++
+----
+$ gpg --verify <signature_name> <1>
+gpg: Signature made Tue 24 Sep 2019 09:38:24 AM PDT using RSA key ID F21541EB
+gpg: Good signature from "Red Hat, Inc. (beta key 2) <security@redhat.com>"
+gpg:                 aka "Mark Cox Internal RSA 4096 test key <mjc@redhat.com>"
+gpg: WARNING: This key is not certified with a trusted signature!
+gpg:          There is no indication that the signature belongs to the owner.
+Primary key fingerprint: B08B 659E E86A F623 BC90  E8DB 938A 80CA F215 41EB
+----
+<1> Specify the name of the signature that you stored.
+
+.. Confirm that `Primary key fingerprint` value from the signature output is
+is listed on the
+link:https://access.redhat.com/security/team/key[Product Signing Keys]
+page on the Customer Portal.
+
 . Mirror the repository:
 +
 ----
 $ oc adm -a ${LOCAL_SECRET_JSON} release mirror \
      --from=quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE} \
      --to=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY} \
-     --to-release-image=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${OCP_RELEASE}
+     --to-release-image=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${DIGEST}
```

Comment:
In the following cluster upgrade command, we decide to use release_version, then here we should NOT mirror release payload image with DIGEST

Reply:
Yeah, we need a tag here to keep the mirrored release from being garbage collected.